test/core/end2end/fixtures/h2_uds.cc

```diff
@@ -46,8 +46,7 @@ struct fullstack_fixture_data {
 
 static int unique = 1;
 
-static grpc_end2end_test_fixture chttp2_create_fixture_fullstack(
-    grpc_channel_args* /*client_args*/, grpc_channel_args* /*server_args*/) {
+static grpc_end2end_test_fixture chttp2_create_fixture_fullstack_base(const std::string& addr) {
   fullstack_fixture_data* ffd = new fullstack_fixture_data;
   ffd->localaddr = absl::StrFormat("unix:/tmp/grpc_fullstack_test.%d.%d",
```

Comment:
I think this should say `ffd->localaddr = std::move(localaddr);`.  Otherwise, we're not actually testing the abstract socket at all.

Reply:
Oops! Right. Done.